doc/releases/release-notes-3.6.rst

```diff
@@ -154,10 +159,19 @@ Boards & SoC Support
 
 * Made these changes for ARM boards:
 
+  * Added support for low power on G1120B0MIPI using RT595
+  * Added support for lpspi, lpi2c on NXP board: ``mimx93_evk_a55``
+  * lpcxpresso55s69: Fix partition naming to use the standard slot naming used by TFM-enabled
+    Zephyr platforms.
+  * Enabled support for linkserver debugger on ``frdm_kl25z``, ``mimxrt1015_evk``,
+    ``mimxrt1020_evk``, ``mimxrt1050_evk``, ``mimxrt685_evk``, ``frdm_k64f``
+  * NXP: Enabled :ref:`linkserver<linkserver-debug-host-tools>` to be the default runner,
+    if none was set in the board's ``board.cmake`` file.
```

Comment:
Didn't linkserver become the default runner on all supported boards?

Reply:
It was made the default for `west` command in this release.

Reply:
Moved this portion to the migration guide.